server/utils/get_request_data_csv.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import requests
+import argparse
+
+REQUESTS_BATCH_SIZE = 10000
+
+def get_311_request_data(start_date, end_date):
+    """Fetches 311 request from the 311 data server. 
+    
+    Retreives 311 requests from the 311 data server for a given start_date and end_date.

```

Comment:
We should also note here that we are writing it to a csv file.

Actually, I would prefer that this function does not write it to a csv file, and that we perform the csv writing in `main`. That way, the purpose of the function is clear and singular.

Reply:
Made this change.